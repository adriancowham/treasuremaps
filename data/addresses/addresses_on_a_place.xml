<posts>
<item rdf:about="http://sacramento.craigslist.org/gms/2339949554.html"> 
<title><![CDATA[moving sale (1390 craydon place eldorado hills)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339949554.html</link> 
<description><![CDATA[Furniture.  Housewares. Men's clothes. Kidstuff.  Artworks.  Craft supplies.  Bric-a-brac.  Exercise equipment. Piano.... etc.
<br> 

<br> 
Opens at 8:30 a.m.  Saturday 4/23
<br> 

<br> 
House has been sold and everything must go.   Something for everyone and everything priced to move!
<br> 

<br> 
916 933-2865
<br> 

<br> 
1390 Craydon Place, Eldorado Hills, 95762
<br> 

<br> 

<br> 

<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=1390 craydon place eldorado hills -->Location: 1390 craydon place eldorado hills
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T21:08:49-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339949554.html</dc:source> 
<dc:title><![CDATA[moving sale (1390 craydon place eldorado hills)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T21:08:49-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339931764.html"> 
<title><![CDATA[huge friday baby yard sale 4-22 (elk Grove)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339931764.html</link> 
<description><![CDATA[we are having a yard sale 
<br> 

<br> 
8am-3pm , 9252 Faraway Place, 647-66662
<br> 

<br> 
cinderella doll house 
<br> 

<br> 
play pen blue with animals $25.00
<br> 

<br> 
green plaid stroller with car seat, base and extras $100.00
<br> 

<br> 
clothes gilrs size 3- 5
<br> 

<br> 
costumes $3.00
<br> 
dresses $3.00 
<br> 
new items $5.00/dresses
<br> 

<br> 
collector barbies in box $5.00 to $10.00
<br> 

<br> 
 princess bike $5.00
<br> 

<br> 
car speaker 6 by 9 new $30.00
<br> 

<br> 
crib with matress and dresser
<br> 

<br> 

<br> 
lots of great stuff come by<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=elk Grove -->Location: elk Grove
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T20:51:36-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339931764.html</dc:source> 
<dc:title><![CDATA[huge friday baby yard sale 4-22 (elk Grove)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T20:51:36-07:00</dcterms:issued> 
</item> 
</posts>