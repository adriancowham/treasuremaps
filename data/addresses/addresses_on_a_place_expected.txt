1390 craydon place
9252 Faraway Place
136 Skyview Place